lstchain/scripts/lstchain_data_muon_analysis.py

```diff
@@ -91,78 +89,57 @@ def main():
                          'impact_x_array': [],
                          'impact_y_array': [],
                          }
-    
+
+    tel_id = args.tel_id
+
     # Calibration related quantities
     r0calib = LSTR0Corrections(
         pedestal_path = args.pedestal_file,
-        r1_sample_start=2,r1_sample_end=38)
-
-    ff_data = FlatFieldContainer()
-    cal_data =  WaveformCalibrationContainer()
-    ped_data =  PedestalContainer()
+        r1_sample_start=2,r1_sample_end=38, tel_id=tel_id)
 
-    dc_to_pe = []
-    ped_median = []
+    charge_config = Config({
+        "LocalPeakWindowSum": {
+            "window_shift": 4,
+            "window_width": 8
```

Comment:
why using here different parameters as those used by default in the r0_to_dl1 analysis?

Reply:
Linked to #208

Reply:
Sorry, actually I think I changed them looking at the waveforms (they seemed the best one).
We should indeed used the values inside a configuration file. Which value are used in the r0_to_dl1 (where are the configuration files?)
But the way, are muons pulses as shower pulses?

Reply:
Hi Franca.
Maybe we can take this opportunity to work on the merging of config files (in another PR) to get a single one?
The one used for `r0_to_dl1` is there: https://github.com/cta-observatory/cta-lstchain/blob/master/lstchain/data/lstchain_standard_config.json

Reply:
Hi @vuillaut,
ok, I will have a look, but I am going before to insert the time pulse correction in the calibration chain. Then, let do a bit of cleaning also in this part.